.eslintrc.json

```diff
@@ -18,7 +18,8 @@
     "@typescript-eslint/interface-name-prefix": "off",
     "comma-dangle": "off",
     "sort-keys": "off",
-    "max-classes-per-file": "off"
+    "max-classes-per-file": "off",
+    "no-use-before-define": "off"
```

Comment:
Why do we want to set this one off?

Reply:
This one makes no sense for most cases - e.g. type definitions - it is always okay to use a type definition before declaration, and also function declaration statements. The TDD (Temporal Dead Zone) detection is much better in TS compiler for the relevant ones. So, this rule in practice was just annoyance rather than preventing any potential bugs.